cmd/ci-operator-prowgen/main.go

```diff
@@ -637,6 +630,23 @@ func generatePeriodicForTest(name string, info *prowgenInfo, label jc.ProwgenLab
 	}
 }
 
+func deplicatePresubmit(info *prowgenInfo, presubmit *prowconfig.Presubmit) *prowconfig.Presubmit {
```

Comment:
nit: this might be better named as `duplicatePresubmitForBuildCluster` or similar

Reply:
Done.